문제1.py

```diff
@@ -0,0 +1,19 @@
+T = int(input())
+for i in range(T):
+    blank = input()
+    b={}
+    d=[]
+    c= input().split()
+    for char in c:
+        b[int(char)]=0
+    for char in c:
+        b[int(char)]+=1
+    e = max(b.values())
+    for k, v in b.items():
+        if v == e:
+            d.append(k)
+    f = max(d)
+    if len(d) == 1:
+        print(f"#{i+1} {d[0]}")
+    else:
+        print(f"#{i+1} {f}")
```

Comment:
정말 좋은 풀이네요! 👍 
```suggestion
    f = max(d)
    print(f"#{i+1} {f}")
```
`f`를 이미 구하셨으므로 따로 조건을 나누지 않고 바로 답을 출력하셔도 될 것 같습니다!